.emacs

```diff
@@ -67,6 +67,7 @@
 
 ;; Write customizations to ~/.emacs.d/custom.el instead of this file.
 (setq custom-file "~/.emacs.d/custom.el")
+(load custom-file t t)
```

Comment:
Thanks, I ran into this same issue and was about to report the issue, but then I saw your pull request.

I vote against the suppression of loading messages, since they help with debugging.  For example, I noticed this issue in my `*Messages*` buffer just now:

```
Loading /xxxx/.emacs.d/custom.el (source)...
Loading winner...done
Problems while trying to load feature ‘drill’
Loading /xxxx/.emacs.d/custom.el (source)...done
```

With the messages it is very clear that my `custom.el` is causing winner to be loaded, and something called 'drill is not loading properly.

I also suggest not hard coding ~/.emacs.d into the config, since there is already a variable for that.

In all, this is what I have:

```
(setq custom-file (concat user-emacs-directory "custom.el"))
(load custom-file 'noerror)
```

Reply:
Good point about `'noerror`, but I'd use `(expand-file-name "custom.el" user-emacs-directory)` instead of `concat`.

I'll update my PR, thanks! :)